homeassistant/components/automation/device.py

```diff
@@ -13,8 +13,14 @@
 )
 
 
+async def async_validate_trigger_config(hass, config):
+    """Validate config.
+
+    This method is a coroutine.
```

Comment:
No need to add this comment. We only add this if it's a non async method that is returning a coroutine

Reply:
OK, fixed.